README.md

```diff
@@ -30,9 +30,11 @@ Connector runtimes provided by [Camunda out-of-the-box](https://docs.camunda.io/
+| RabbitMQ     | [Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-rabbitmq)     | [GitHub](https://github.com/camunda/connector-rabbitmq)     | [Camunda Docs](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/rabbitmq/)    | [Camunda Platform Self-Managed Free Edition] | [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-) |
 | REST         | [Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-http-json)    | [GitHub](https://github.com/camunda/connector-http-json)    | [Camunda Docs](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/rest/)        | [Apache 2.0]                                 | [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-) |
 | SendGrid     | [Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-sendgrid)     | [GitHub](https://github.com/camunda/connector-sendgrid)     | [Camunda Docs](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid/)    | [Camunda Platform Self-Managed Free Edition] | [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-) |
 | Slack        | [Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-slack)        | [GitHub](https://github.com/camunda/connector-slack)        | [Camunda Docs](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/slack/)       | [Camunda Platform Self-Managed Free Edition] | [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-) |
+| SNS          | [Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-sns)          | [GitHub](https://github.com/camunda/connector-sns)          | [Camunda Docs](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/sns/)         | [Camunda Platform Self-Managed Free Edition] | [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-) |
```

Comment:
Let's go with "Amazon SNS" instead of "SNS" and put it above SQS, so we're consistent with SQS.